dashboard/app/controllers/abuse_report_controller.rb

```diff
@@ -0,0 +1,43 @@
+require 'json'
+require 'httparty'
+
+class AbuseReportController < ApplicationController
```

Comment:
Does it make sense to have our own controller for this, or is there an existing controller it would make sense to put this logic in?